rasa/core/policies/policy.py

```diff
@@ -114,17 +121,37 @@ def _create_featurizer(
 
     def __init__(
         self,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
         featurizer: Optional[TrackerFeaturizer] = None,
```

Comment:
I kept the featurizer separate as its type changes (in the config it's a `Dict` and then we later instantiate it)